include/psa/crypto_se_driver.h

```diff
@@ -980,12 +980,12 @@ typedef struct {
      * Use #PSA_DRV_SE_HAL_VERSION.
      */
     uint32_t hal_version;
-    psa_drv_se_key_management_t key_management;
-    psa_drv_se_mac_t mac;
-    psa_drv_se_cipher_t cipher;
-    psa_drv_se_aead_t aead;
-    psa_drv_se_asymmetric_t asymmetric;
-    psa_drv_se_key_derivation_t derivation;
+    const psa_drv_se_key_management_t *key_management;
```

Comment:
Consider also making the pointer itself const, if possible. Without that, you can't get any of that sweet armclang LTO action.

Reply:
Isn't it enough to make the whole structure `const`?

Reply:
Needs testing.

Reply:
I'm not planning to change anything here in this PR. It's a more general issue for the driver interface anyway. If you think a change is needed, please file it separately in https://github.com/ARMmbed/psa-crypto/issues.